$OpenBSD: patch-lib_Mail_DMARC_Report_Send_pm,v 1.1 2016/04/29 12:54:02 giovanni Exp $

Don't look in $PWD/lib, it breaks startup if $PWD is inaccessible by the
current user.

--- lib/Mail/DMARC/Report/Send.pm.orig	Tue Apr 26 11:38:25 2016
+++ lib/Mail/DMARC/Report/Send.pm	Tue Apr 26 11:38:31 2016
@@ -3,7 +3,6 @@ our $VERSION = '1.20150908'; # VERSION
 use strict;
 use warnings;
 
-use lib 'lib';
 use parent 'Mail::DMARC::Base';
 use Mail::DMARC::Report::Send::SMTP;
 use Mail::DMARC::Report::Send::HTTP;
